.github/workflows/config/yamllint_config.yml

```diff
@@ -8,7 +8,7 @@ yaml-files:
   - '.yamllint'
 
 ignore: |
-  **/templates/*.yaml
```

Comment:
This should be two entries so you don't break linting for existing charts:
  **/templates/*.yaml
  **/templates/**/*.yaml

Reply:
fixed